src/app/core/components/left-menu/left-menu.component.scss

```diff
@@ -33,6 +33,60 @@ $archive-options-shadow-offset: 0.5rem;
     width: $left-menu-width + 4 * $grid-unit;
   }
 
+  hr {
+    width: 180px;
+    height: 1px;
+    margin: 0 auto;
+    border-top: solid 1px rgba(255, 255, 255, 0.15);
+  }
+
+  .apps-menu {
+    width: 100%;
+    display: flex;
+    height: 3 * $grid-unit;
+    flex-direction: row;
+    justify-content: space-between;
+
+    .apps-link-main {
+      width: 75%;
+      height: 3 * $grid-unit;
+      display: flex;
+      align-items: center;
+
+      i, img.svg-icon {
+        flex: 0 0 $left-menu-icon-width;
+        font-size: $mat-icon;
+        text-align: center;
+      }
+
+      span {
+        height: 3 * $grid-unit;
+        flex: 1;
+        display: flex;
+        align-items: center;
+      }
+    }
+
+    .apps-toggle {
+      padding-right: 12px;
+      margin-right: 20px;
+      height: 3 * $grid-unit;
+      display: flex;
+      align-items: center;
+
+      i {
+        font-size: $mat-icon-lg;
+        transition: transform 250ms $tweaked-ease;
```

Comment:
This is very fun! Amazing how rewarding a tiny bit of animation can feel. :slightly_smiling_face: